docs/components/table-view.md

```diff
@@ -922,6 +1093,47 @@ Slot props:
 </KTableView>
 ```
 
+### row-expanded
+
+Slot for passing custom content that will be revealed once user expands one of the table rows when [`expandableRows` prop](#expandablerows) is `true`.
+
+Slot props:
+
+* `row` - table row object
+* `rowKey` - table row index
```

Comment:
question: what’s the purpose of exposing an index? Child data likely wouldn’t have any knowledge of the parent, but even so, what would the index of the parent be used for?

Reply:
I'm not sure what application would it have but we are exposing the same property in cell slots - maybe there's usage of it somewhere.

Reply:
I’m not sure children would have a need for the parent’s index?

Reply:
Okay, removing this slot prop.